src/figuresClasses.ts

```diff
@@ -1,11 +1,84 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  constructor(
+    public color: string,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    if (this.a <= 0 || this.b <= 0 || this.c <= 0) {
+      throw new Error('your error message');
+    }
+
+    const arrOfSides = [a, b, c];
+    const maxSide = Math.max(...arrOfSides);
+    const smallerSidesSum = arrOfSides
+      .filter((item) => item < maxSide)
+      .reduce((acc, currentValue) => acc + currentValue);
+
+    if (maxSide >= smallerSidesSum) {
+      throw new Error('your error message');
```

Comment:
The error message 'your error message' is not descriptive. Consider providing a more informative message, such as 'The sum of the lengths of any two sides must be greater than the length of the third side.'